web/admin/src/App.css

```diff
@@ -216,7 +216,6 @@ html, body, input, button, select, form{
 
 .grayText{
   color: #AEAEAE;
-  padding: 0px;
```

Comment:
Why the removal?

Reply:
It did not resolve the original CSS bug and should not have been included in my first PR